### PR TITLE
add sfn coursebooks

### PR DESCRIPTION
**Pull request by loftusa**
coursebooks for sfn 2017 and 2018.

### CODE REVIEW

File: content/pubs/pubs.bib [hunk 1]
```diff
@@ -1,3 +1,17 @@
+@article{sfn2018chapter,
+  title={Graph Classification Using Signal-Subgraphs: Applications in Statistical Connectomics},
+  author={Vogelstein, Joshua T},
+  journal={SfN 2018 course book},
+  year={2018},
+  url={https://neurodata.io/talks/sfn_2018_coursebook.pdf},
+}
+@article{sfn2017chapter,
+  title={What is Connectome Coding?},
```
Comment (falkben) [inline content/pubs/pubs.bib]: I believe you have the articles mixed up between 2017/2018.

File: content/pubs/pubs.bib [hunk 2]
```diff
@@ -1,3 +1,17 @@
+@article{sfn2018chapter,
+  title={Graph Classification Using Signal-Subgraphs: Applications in Statistical Connectomics},
+  author={Vogelstein, Joshua T},
```
Comment (falkben) [inline content/pubs/pubs.bib]: Looks like you are missing the other authors.

**Review by falkben — CHANGES_REQUESTED** [review]
Bib entries

File: content/pubs/pubs.bib (hunk 1 above)
Comment (loftusa) [reply content/pubs/pubs.bib]: Thanks! Fixed

File: content/pubs/pubs.bib (hunk 2 above)
Comment (loftusa) [reply content/pubs/pubs.bib]: Added, nice catch.

**Review by falkben — APPROVED** [review]

**Pull request merged**